packages/connect-core/src/entities/App.ts

```diff
@@ -61,6 +61,19 @@ export default class App {
     return this.organization.connection.orgConnector
   }
 
+  contract(): ethers.Contract {
```

Comment:
I include a helper to have access to an Ethers contract object to make it super easy to interact with the blockchain for the particular App.